CONTRIBUTING.md

```diff
@@ -0,0 +1,628 @@
+                "arn:aws:iot:*:*:stream/*"
+            ]
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": "ses:SendEmail",
+            "Resource": "arn:aws:ses:us-east-1:XXXXXXXXXXX:identity/XXXXXXXXXXX:"
+        },
+        {
+            "Sid": "VisualEditor2",
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:DescribeAccountLimits",
+                "cloudwatch:GetMetricStatistics",
+                "cloudwatch:ListMetrics",
+                "comprehend:BatchDetectEntities",
+                "comprehend:DetectDominantLanguage",
+                "comprehend:DetectEntities",
+                "comprehend:DetectKeyPhrases",
+                "comprehend:DetectSentiment",
+                "connect:*",
+                "dynamodb:*",
+                "ec2:DescribeImages",
+                "ec2:DescribeInstances",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "firehose:ListDeliveryStreams",
+                "iot:*",
+                "kinesis:ListStreams",
+                "kinesisvideo:ListFragments",
+                "kinesisvideo:ListStreams",
+                "kms:CreateKey",
+                "lambda:ListFunctions",
+                "mobiletargeting:PutEvents",
+                "mobiletargeting:UpdateEndpoint",
+                "polly:DeleteLexicon",
+                "polly:GetSpeechSynthesisTask",
+                "polly:PutLexicon",
+                "polly:StartSpeechSynthesisTask",
+                "polly:SynthesizeSpeech",
+                "s3:*",
+                "sdb:ListDomains",
+                "ses:GetSendQuota",
+                "ses:VerifyEmailIdentity",
+                "sns:GetEndpointAttributes",
+                "sns:ListTopics",
+                "sqs:ListQueues",
+                "textract:AnalyzeDocument",
+                "transcribe:*",
+                "translate:TranslateText"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor3",
+            "Effect": "Allow",
+            "Action": "elasticloadbalancing:ConfigureHealthCheck",
+            "Resource": "arn:aws:elasticloadbalancing:*:*:loadbalancer/*"
+        },
+        {
+            "Sid": "IoTStreamingTest",
+            "Effect": "Allow",
+            "Action": "iam:PassRole",
+            "Resource": "arn:aws:iam::XXXXXXXXXXX:role/OTAIntegTest"
+        }
+    ]
+}
+```
+
+**Policy 2**
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "mobileanalytics:PutEvents",
+                "cognito-sync:*"
+            ],
+            "Resource": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+If you are looking to run integration tests inside the Auth project, you will need to create a `awsconfiguration.json` file in AWSAuthSDK/AWSMobileClientTests that looks like this:
```

Comment:
building on my previous comment about having this informatiion where they need it, we could have a README at the root of AWSAuthSDK/AWSMobileClientTests.

Reply:
ill move these files to where they need to go and refer back to them

Reply:
will do in a v2